### PR TITLE
fix 296 US airports geom(from Wikidata) +concordances(Wikidata+OurAir…

### PR DESCRIPTION
**Pull request by ImreSamu**
please review ! and check !

updated: 296 US airports geom + concordances

input:
* wikidata    (~2018-09-17)
* ourairports (~2018-11-01)  ( http://ourairports.com/airports.csv )

selection - for the 296 US airports:  
* wof:name == wikidata_name  ( 'OK-ADD:suggested for add-N1Full-name-match' )
* and wikidata:geom inside in the wof(parent->geom )
* and wikidata:country = Q30 ( US ) 
* and has wd(icao) 
* and oa:iata = wd:iata and oa:faa=wd:faa   ( OurAirports & Wikidata has a same info)

Added/updated:
* `oa:elevation_ft`
* `oa:type`
* `geom:latitude`   ( from wikidata )
* `geom:longitude`  ( from wikidata )
* `geometry`        ( from wikidata )
* `src:geom`: "wikidata"
* `wof:concordances`
  * `faa:code`
  * `iata:code`
  * `icao:code`
  * `oa:id`
  * `wd:id`
  * `wk:page`  (only fix)

Not Now:
* Checking/Importing/fixing  names

importCode/csv
* https://gist.github.com/ImreSamu/5609656faac83b1fad6b38ff2291a5e1

Audit github markdown - about the changed values
( iata manual check: https://www.iata.org/publications/Pages/code-search.aspx )

 event  |  id       |  property | old value| new value |
------- |-----------|-----------|----------|-----------|
|update|[102532379](https://spelunker.whosonfirst.org/id/102532379/)|`wk:page`|Princeton Municipal Airport (Minnesota)|Princeton Municipal Airport (Maine)|
|remove|[102532643](https://spelunker.whosonfirst.org/id/102532643/)|`iata:code`|EZM||
|remove|[102526885](https://spelunker.whosonfirst.org/id/102526885/)|`iata:code`|TZV||
|remove|[102529627](https://spelunker.whosonfirst.org/id/102529627/)|`iata:code`|AOV||
|remove|[102532567](https://spelunker.whosonfirst.org/id/102532567/)|`iata:code`|HXF||
|remove|[102533421](https://spelunker.whosonfirst.org/id/102533421/)|`iata:code`|OLG||
|remove|[102531409](https://spelunker.whosonfirst.org/id/102531409/)|`iata:code`|UBE||
|remove|[102526373](https://spelunker.whosonfirst.org/id/102526373/)|`iata:code`|AZC||
|remove|[102531479](https://spelunker.whosonfirst.org/id/102531479/)|`iata:code`|HYW||
|remove|[102531351](https://spelunker.whosonfirst.org/id/102531351/)|`iata:code`|CQX||
|update|[102531291](https://spelunker.whosonfirst.org/id/102531291/)|`iata:code`|JSO|JKV|
|remove|[102525717](https://spelunker.whosonfirst.org/id/102525717/)|`iata:code`|DCY||
|remove|[102528849](https://spelunker.whosonfirst.org/id/102528849/)|`iata:code`|TSO||
|remove|[102531601](https://spelunker.whosonfirst.org/id/102531601/)|`iata:code`|CFJ||
|remove|[102528771](https://spelunker.whosonfirst.org/id/102528771/)|`iata:code`|EPM||
|remove|[102529783](https://spelunker.whosonfirst.org/id/102529783/)|`iata:code`|GGI||
|remove|[102534347](https://spelunker.whosonfirst.org/id/102534347/)|`iata:code`|CTK||
|remove|[102534063](https://spelunker.whosonfirst.org/id/102534063/)|`iata:code`|HCD||
|remove|[102533999](https://spelunker.whosonfirst.org/id/102533999/)|`iata:code`|HVC||
|remove|[102534087](https://spelunker.whosonfirst.org/id/102534087/)|`iata:code`|AXH||
|remove|[102534113](https://spelunker.whosonfirst.org/id/102534113/)|`iata:code`|HQG||
|remove|[102530783](https://spelunker.whosonfirst.org/id/102530783/)|`iata:code`|GVQ||
|remove|[102526287](https://spelunker.whosonfirst.org/id/102526287/)|`iata:code`|GPH||
|update|[102531113](https://spelunker.whosonfirst.org/id/102531113/)|`iata:code`|RVL|RED|
|remove|[102525365](https://spelunker.whosonfirst.org/id/102525365/)|`iata:code`|EUL||
|remove|[102532575](https://spelunker.whosonfirst.org/id/102532575/)|`iata:code`|HBZ||
|remove|[102533053](https://spelunker.whosonfirst.org/id/102533053/)|`iata:code`|AFE||
|remove|[102529309](https://spelunker.whosonfirst.org/id/102529309/)|`iata:code`|BVN||
|update|[102528757](https://spelunker.whosonfirst.org/id/102528757/)|`iata:code`|IZG|FRY|
|remove|[102530729](https://spelunker.whosonfirst.org/id/102530729/)|`iata:code`|FWS||
|remove|[102528277](https://spelunker.whosonfirst.org/id/102528277/)|`iata:code`|GGF||
|update|[102527663](https://spelunker.whosonfirst.org/id/102527663/)|`iata:code`|HSE|HNC|
|update|[102534275](https://spelunker.whosonfirst.org/id/102534275/)|`iata:code`|RZL|RNZ|
|update|[102533673](https://spelunker.whosonfirst.org/id/102533673/)|`iata:code`|PGU|PGA|
|update|[102533673](https://spelunker.whosonfirst.org/id/102533673/)|`icao:code`|OIBI|KPGA|
|remove|[102526197](https://spelunker.whosonfirst.org/id/102526197/)|`iata:code`|LHX||
|update|[102532083](https://spelunker.whosonfirst.org/id/102532083/)|`iata:code`|RAD|RRT|
|remove|[102529387](https://spelunker.whosonfirst.org/id/102529387/)|`iata:code`|PQN||
|update|[102533317](https://spelunker.whosonfirst.org/id/102533317/)|`iata:code`|HHW|HUJ|
|remove|[102533281](https://spelunker.whosonfirst.org/id/102533281/)|`iata:code`|HSD||
|remove|[102532219](https://spelunker.whosonfirst.org/id/102532219/)|`iata:code`|AHQ||
|remove|[102532101](https://spelunker.whosonfirst.org/id/102532101/)|`iata:code`|ACQ||
|update|[102532119](https://spelunker.whosonfirst.org/id/102532119/)|`iata:code`|AFJ|WSG|
|remove|[102532237](https://spelunker.whosonfirst.org/id/102532237/)|`iata:code`|EGT||
|remove|[102532177](https://spelunker.whosonfirst.org/id/102532177/)|`iata:code`|CTJ||

(for me)
the 296 record selected with this SQL filter: 
```sql
     wd_p239_icao is not null 
 and new_p17_country_id = '[{"Q30": "United States of America"}]' 
 and _matching_category = 'OK-ADD:suggested for add-N1Full-name-match'
 and ndups_suggestion is null
 and nDups is null
 and base_geom_type='GeomMissing'
 and a_comp_id_distance ='{0}'
 and      (a_wof_type  @> ARRAY['campus'] )
 and  not (a_wof_type  @> ARRAY['wof'] )
 and  not (a_wof_type  @> ARRAY['locality'] ) 
 and not new_is_cebauno
 and coalesce(oai.local_code,'') = coalesce(arp.wd_p240_faa,'')
 and coalesce(oai.iata_code,'')  = coalesce(arp.wd_p238_iata,'')
```
;

### CODE REVIEW

File: data/102/521/833/102521833.geojson [hunk 1]
```diff
@@ -6,13 +6,15 @@
...
-    "geom:latitude":0.0,
-    "geom:longitude":0.0,
+    "geom:bbox":"-88.5291666667,30.4627777778,-88.5291666667,30.4627777778",
+    "geom:latitude":30.462778,
+    "geom:longitude":-88.529167,
     "iso:country":"",
     "mz:hierarchy_label":1,
     "mz:is_current":1,
-    "src:geom":"missing",
+    "oa:elevation_ft":"17",
```
Comment (stepps00) [inline data/102/521/833/102521833.geojson]: noting that this is correct, we store this value as a string: https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/oa/elevation_ft.json

**Comment by stepps00** [comment]
I spot checked various properties and geometries, looking good!

**Comment by ImreSamu** [comment]
I have tested the name differences with OurAirports (   wof_name  vs.  oa_name  )
* [Canby Municipal Airport](https://en.wikipedia.org/wiki/Canby_Municipal_Airport)  also known as `Myers Field`


|       [ _jw](https://en.wikipedia.org/wiki/Jaro%E2%80%93Winkler_distance)        |                    wof_name                     |                      oa_name                       |  wof_id   | new_wd_id |
|-------------------|-------------------------------------------------|----------------------------------------------------|-----------|-----------|
|  0.49064558629776 | Canby Municipal Airport                         | Myers Field                                        |[102522307](https://spelunker.whosonfirst.org/id/102522307)|[Q14692549](https://www.wikidata.org/wiki/Q14692549)|
| 0.643163550397593 | Naval Air Station Joint Reserve Base Fort Worth | NAS Fort Worth JRB/Carswell Field                  |[102524719](https://spelunker.whosonfirst.org/id/102524719)|[Q3016367 ](https://www.wikidata.org/wiki/Q3016367 )|
| 0.864076858813701 | Yolo County Airport                             | Yolo County Davis Woodland Winters Airport         |[102530963](https://spelunker.whosonfirst.org/id/102530963)|[Q10853689](https://www.wikidata.org/wiki/Q10853689)|
| 0.885698924731183 | Ontonagon County Airport                        | Ontonagon County Schuster Field                    |[102533647](https://spelunker.whosonfirst.org/id/102533647)|[Q27044134](https://www.wikidata.org/wiki/Q27044134)|
| 0.886517615176152 | Kerrville Municipal Airport                     | Kerrville Municipal Louis Schreiner Field          |[102533055](https://spelunker.whosonfirst.org/id/102533055)|[Q6394435 ](https://www.wikidata.org/wiki/Q6394435 )|
|             0.888 | Troy Municipal Airport                          | Troy Municipal Airport at N Kenneth Campbell Field |[102527761](https://spelunker.whosonfirst.org/id/102527761)|[Q7846908 ](https://www.wikidata.org/wiki/Q7846908 )|
| 0.890865800865801 | Sidney Municipal Airport                        | Sidney Municipal-Lloyd W Carr Field                |[102526617](https://spelunker.whosonfirst.org/id/102526617)|[Q7509249 ](https://www.wikidata.org/wiki/Q7509249 )|
| 0.904347826086957 | Cheraw Municipal Airport                        | Cheraw Municipal Airport/Lynch Bellinger Field     |[102531279](https://spelunker.whosonfirst.org/id/102531279)|[Q1708066 ](https://www.wikidata.org/wiki/Q1708066 )|
| 0.908370173887415 | West Georgia Regional Airport                   | West Georgia Regional O V Gray Field               |[102532177](https://spelunker.whosonfirst.org/id/102532177)|[Q7985208 ](https://www.wikidata.org/wiki/Q7985208 )|
| 0.908549783549784 | Hutchinson Municipal Airport                    | Hutchinson Municipal Butler Field                  |[102534063](https://spelunker.whosonfirst.org/id/102534063)|[Q3084732 ](https://www.wikidata.org/wiki/Q3084732 )|


my sql code for selection:
```sql
whosonfirst=# select 
whosonfirst-#   jarowinkler( base_name , name ) as _jw
whosonfirst-#   ,base_name as wof_name
whosonfirst-#   ,name      as oa_name
whosonfirst-#   ,base_id   as wof_id
whosonfirst-#   ,_suggested_comp_id as new_wd_id
whosonfirst-# from newd.airport_usa_01
whosonfirst-# order by jarowinkler( base_name , name ) 
whosonfirst-# limit 10
whosonfirst-# ;
```

**Comment by stepps00** [comment]
I've spot checked a dozen of these records and things are looking good -- thank you for the PR, @ImreSamu!!

**Pull request merged**